cvat-sdk/cvat_sdk/core/client.py

```diff
@@ -167,7 +165,22 @@ def _detect_schema(cls, base_url: str) -> str:
                     if response.status in [200, 401]:
                         # Server versions prior to 2.3.0 respond with unauthorized
                         # 2.3.0 allows unauthorized access
-                        return schema
+                        return True
+            return False
+
+        if attempt("https"):
+            return "https"
+
+        self.logger.warning(
+            "Failed to connect to the server using HTTPS; will attempt HTTP instead"
+        )
+        self.logger.warning(
```

Comment:
Can we keep it only for localhost?

Reply:
I'd rather not; it would make the behavior too magical for my taste. Moreover, for SDK usage I don't think it makes that much difference - if you're writing code, it's not too hard to add `http:`.

I will make the default CLI URL `http://localhost`, though, which should make it easier to use the CLI (I meant to do this to begin with, I just forgot 🙂).